### ISSUE TITLE
LG Air Conditioner - Energy Current 0W or 5W

### ISSUE THREAD

**Issue by danilopeseto** [ollo69/ha-smartthinq-sensors#618]
I have 5 A/C units using this integration.

Model: AMNW18GSKR0

Modem: clip_hna_v1.9.192_RT / clip_hna_v1.9.192_QC
Firmwares: SAA42760917.00033BA.0 / SAA386904409.00000409.0

I have only 2 values on energy.current sensor 5W on standby ou 0W on powered on.

**Comment by ollo69** [comment]
Please complete the issue template in all the parts and attach integration diagnostic.

**Comment by danilopeseto** [comment]
Like this ?

[smartthinq_sensors-1ab00533a818eb8e4b6ca9bb77d1eb65-Ar Varanda-2b07d0eca9e9bd5bdeab491b33aafa2b.json.txt](https://github.com/ollo69/ha-smartthinq-sensors/files/12965108/smartthinq_sensors-1ab00533a818eb8e4b6ca9bb77d1eb65-Ar.Varanda-2b07d0eca9e9bd5bdeab491b33aafa2b.json.txt)

**Comment by Elrendir95** [comment]
Hello I am facing the same issue 

**Describe the bug**
The "Energy current" current sensor showing 5W when it is off or 0W when it is on ( heating )

**Expected behavior**
0W when off some other value when on 

**Screenshots**
![image](https://github.com/ollo69/ha-smartthinq-sensors/assets/85612240/191b36e4-5ae9-4985-b678-17e1dd9a65b2)

**Environment details:**

Environment (HASSIO, Raspbian, etc): TrueNAS Scale Application ( Container )
Home Assistant version installed: 2023.10.5
Component version installed: v0.35.2
Last know working version: unknown ( I have just installed it )
LG device type and model with issue: Displayed in Home Assistant RAC_056905_WW-RAC 
LG devices connected (list): 
3x PM07SP.NSJ
2x PC12SQ.NSJ
1x PC09SQ.NSJ

Debug logs for the polling of one of them if it can help

`2023-11-08 09:50:16.191 DEBUG (MainThread) [custom_components.smartthinq_sensors.wideq.core_async] thinq2_get after: {'resultCode': '0000', 'result': {'appType': 'NUTS', 'modelAppType': 'NUTS', 'modelCountryCode': 'WW', 'countryCode': 'FR', 'modelName': 'RAC_056905_WW', 'deviceType': 401, 'deviceCode': 'AI01', 'alias': 'Clim Bureau', 'deviceId': '7c6a3bdb-e8a4-1aa1-bdef-b8165f8c4889', 'fwVer': '', 'imageFileName': 'ac_home_wall_airconditioner_img.png', 'ssid': 'NETGEAR_EXT', 'softapId': '', 'softapPass': '', 'macAddress': '', 'networkType': '02', 'timezoneCode': 'Europe/Paris', 'timezoneCodeAlias': 'Europe/Paris', 'utcOffset': 1, 'utcOffsetDisplay': '+01:00', 'dstOffset': 2, 'dstOffsetDisplay': '+02:00', 'curOffset': 1, 'curOffsetDisplay': '+01:00', 'sdsGuide': '{"deviceCode":"AI01"}', 'newRegYn': 'N', 'remoteControlType': '', 'userNo': 'FR2005314726928', 'tftYn': 'N', 'deviceState': 'E', 'snapshot': {'airState.windStrength': 6.0, 'airState.wMode.lowHeating': 0.0, 'airState.diagCode': 0.0, 'airState.lightingState.displayControl': 0.0, 'airState.wDir.hStep': 35.0, 'mid': 594949633.0, 'airState.energy.onCurrent': 0.0, 'airState.wMode.airClean': 0.0, 'airState.quality.sensorMon': 0.0, 'airState.tempState.target': 19.0, 'airState.miscFuncState.autoDryRemainTime': 0.0, 'airState.operation': 1.0, 'airState.wMode.jet': 0.0, 'airState.wDir.vStep': 100.0, 'timestamp': 1699433386410.0, 'airState.powerSave.basic': 0.0, 'fwUpgradeInfo': {'upgSched': {'upgUtc': '0', 'cmd': 'none'}}, 'static': {'deviceType': '401', 'countryCode': 'FR'}, 'airState.tempState.current': 19.0, 'airState.miscFuncState.extraOp': 0.0, 'airState.reservation.sleepTime': 0.0, 'airState.miscFuncState.autoDry': 0.0, 'airState.reservation.targetTimeToStart': 0.0, 'meta': {'allDeviceInfoUpdate': False, 'messageId': 'tNlPodCpSoCs07U-QNQEzQ'}, 'online': True, 'airState.opMode': 4.0, 'airState.reservation.targetTimeToStop': 0.0, 'airState.filterMngStates.maxTime': 0.0, 'airState.filterMngStates.useTime': 0.0}, 'online': True, 'platformType': 'thinq2', 'area': 133661, 'regDt': 20230927130139.0, 'blackboxYn': 'Y', 'modelProtocol': 'STANDARD', 'receipeVersion': 0, 'activeSaving': 'OFF', 'smartCareV2': 'OFF', 'order': 0, 'nlpAlias': 'none', 'drServiceYn': 'N', 'fwInfoList': [{'checksum': '00000409', 'order': 1.0, 'partNumber': 'SAA38690409'}], 'brandType': 'common', 'regDtUtc': '20230927110139', 'regIndex': 0, 'groupableYn': 'Y', 'controllableYn': 'Y', 'combinedProductYn': 'N', 'masterYn': 'Y', 'initDevice': False, 'firebaseLogKey': 'P:SD', 'manufacture': {'inventoryOrg': 'EAT', 'macAddress': 'B8:16:5F:8C:48:89', 'manufactureModel': 'S3-M12JA2DA', 'manufacturedAt': '2023-03-22T00:47:39+00:00', 'registeredAt': '2023-03-22T13:05:38.681715+00:00', 'salesModel': 'S3-M12JA2DA', 'serialNo': '303TKWKB5977'}, 'upgradableYn': 'N', 'autoFwDownloadYn': 'N', 'csUpgradableYn': 'N', 'foodPoisonIndex': '0', 'tclcount': 0}}`

Maybe it is not supported by the device ?

**Comment by ollo69** [comment]
Yes, I suppose is not supported. Complex think is correctly detect unsupported features. I will try to manage this issue in future releases if I found a way. For now I suggest you to just disable the sensor.

**Comment by rodolfomachado** [comment]
I have 2 ACs and both always show 5W. 
Maybe 5W is what it consumes when it's off? 
But I always coming to here to post a question and it may be related to this.
I've set a helper to integrate the consume (W) and I'm seeing a big difference between what HA and LG app.

For instance, this is what HA shows for October:
![image](https://github.com/ollo69/ha-smartthinq-sensors/assets/42542786/96f3573b-0089-4eff-bb16-89e9c96868f5)

And here is the app:
![image](https://github.com/ollo69/ha-smartthinq-sensors/assets/42542786/af0ec6ee-d597-4ef8-88e9-7af173fd619b)

Does anyone have a clue of why that difference?
